code/modules/mob/mob.dm

```diff
@@ -1438,6 +1438,8 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 		if(!("\ref[target]" in faction_src))
 			faction_target -= "\ref[target]" //same thing here.
 		return faction_check(faction_src, faction_target, TRUE)
+	if(!target)
+		return faction_check(faction, null, FALSE)
```

Comment:
This should probably go at the top of the proc, since the `exact_match` block works with `target` even though it could be null.

Reply:
Done!